docs/conf.py

```diff
@@ -135,7 +140,10 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'  # use the theme in subdir 'theme'
+
+html_theme = "qiskit_sphinx_theme"
+
+html_theme_path = [".", qiskit_sphinx_theme.get_html_theme_path()]
```

Comment:
I don't think you need to set this (nor to `import qiskit_sphinx_theme` at the top) - Sphinx should automatically find it through its plugin interface.